doc/index.html

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Is this page edited directly in HTML? I don't have a problem with that, just wondering whether there is a markup language source file for the content.